src/Voron/ReadResult.cs

```diff
@@ -16,7 +16,7 @@ public ReadResult(in ValueReader reader) : this(reader.Base, reader.Length) { }
         
         public int ReadLittleEndianInt32OrDefault(int @default = 0) => IsNull ? @default : Reader.ReadLittleEndianInt32();
         
-        public byte ReadByteOrDefault(byte @default = 0) => IsNull ? default : Reader.ReadByte();
+        public byte ReadByteOrDefault(byte @default = 0) => IsNull ? @default : Reader.ReadByte();
```

Comment:
Can we avoid this confusion by using a different arg name?

Reply:
Yes. Will align with https://learn.microsoft.com/pl-pl/dotnet/api/system.nullable-1.getvalueordefault?view=net-8.0#system-nullable-1-getvalueordefault(-0)